cdap-docs/build.sh

```diff
@@ -26,17 +26,9 @@
 # Builds each of these individually, and then packages them into a single zip file for distribution.
 # _common directory holds common files and scripts.
 
+source _common/vars
```

Comment:
Are these vars actually common, or are they only used by the top-level? I think we should keep in common only variable that really are common...no?